common-core-open/src/main/java/edu/isi/nlp/files/KeyValueSources.java

```diff
@@ -76,20 +87,50 @@ private KeyValueSources() {
    */
   @Nonnull
   public static ImmutableKeyValueSource<Symbol, ByteSource> fromZip(
-      final ZipFile zipFile, final Function<String, Symbol> idExtractor) {
```

Comment:
Because `ZipKeyValueSource` already stores a mapping from key IDs to zip entry names, I think it would be cleaner to drop the `entryFilter` argument being introduced here and just have a `fromVistaUtilsZipKeyValueSource` static method which reads `__keys` and explicitly constructs the second argument to the `ZipKeyValueSource` constructor.